source/flow.rst

```diff
@@ -234,9 +234,8 @@ Ftp request
 
 This node sends a file to a FTP server. When uploading a
 file, its name can be set by setting the "Filename" field in the same way as
```

Comment:
`can be set by setting the "Filename" field` -> `can be set by the "Filename" field`